packages/owl-bot/src/configs-store.ts

```diff
@@ -97,5 +122,39 @@ export interface ConfigsStore {
    */
   findReposAffectedByFileChanges(
     changedFilePaths: string[]
-  ): Promise<GithubRepo[]>;
+  ): Promise<AffectedRepo[]>;
+}
+
+/**
+ * Examines the contents of a local repo directory and collects owl bot config
+ * files.
+ */
+export function collectConfigs(
+  dir: string
+): [OwlBotLock | undefined, OwlBotYamlAndPath[]] {
+  let lock: OwlBotLock | undefined;
+  const yamls: OwlBotYamlAndPath[] = [];
+
+  // .OwlBot.lock.yaml is always in a known location.
+  const lockPath = path.join(dir, owlBotLockPath);
+  if (fs.existsSync(lockPath)) {
+    const lockText = fs.readFileSync(lockPath, 'utf8');
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const lockYaml = load(lockText) as Record<string, any>;
```

Comment:
nit: is it necessary to use any here? Would it be possible to create an interface instead?

Reply:
I haven't found a better alternative to using any here.  :-/